src/handbook/peopleops/leave.md

```diff
@@ -20,7 +20,7 @@ Before you can take time off you should _always_:
 FlowFuse has a unlimited time off policy. Taking vacation is encouraged for all
 team members. To prevent an undefined number of vacation days to start a race to
 the bottom, we recommend each team member to take a minimum of 25 days a year,
-and at least 5 days a quarter.
+and at least 5 days a quarter. It's important to note that although we support unlimited paid time of, we greatly appreciate it when employees communicate their planned time off to their team members. This helps ensure efficient workload distribution and collaboration within the team.
```

Comment:
So it's in the calendar already, what's this line for?

Reply:
We need this line documented in the handbook irrespective of it being in the calendar to avoid any sort of issues at a later stage especially since we provided unlimited PTO